apps/testingaccessibility/README.md

```diff
@@ -189,17 +191,71 @@ The `dev:stripe` node script is a shorthand for `stripe listen --forward-to loca
 ### Connect to the Planetscale database
 
 ```bash
-pscale connect testing-accessibility next-steps --port 3309
+pscale connect testing-accessibility PLANETSCALE_DB_BRANCH_NAME --port 3309
+```
+
+### Changing the Database Schema
+
+**Before making any changes to the schema, consider the impact those changes will have on a production system with preexisting live data.** You will likely need to take a multi-step approach to get to your desired schema. [Prisma's Expand-and-Contract Pattern guide](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern) is a good reference for what this multi-step approach might look like.
+
+To alter the schema, you'll need to first update `/packages/database/prisma/schema.prisma`.
+
+Worth noting if you're familiar with writing migrations for database updates: With Planetscale, you alter the schema directly instead of writing migrations. The upside of this approach is that it leaves the responsibility of safe migrations up to Planetscale. The downside is that it will frequently require multiple separate releases if following the Expand-and-Contract Pattern.
+
+#### Migrate Schema Changes Locally
+
+When you make changes to the schema via Prisma you'll need to explicitly push it to the DB. We don't use Prisma migrations.
+
+You can test out your schema changes locally with the MySQL in Docker instance.
+
+First, make sure Docker is running with MySQL at the appropriate port (`3309`).
+
+Then, apply the changes to the MySQL databse using Prisma's `db push` command.
+
+```bash
+npx prisma db push
+```
+
+From here, you can connect to the MySQL database with a SQL client to view the changes or view it from _Prisma Studio_.
+
+#### Develop against the New Schema
+
+To develop against the new schema with the full power of Prisma's TypeScript typings, you'll need to re-generate the Prisma client for the updated schema. When doing this, if you switch partner products, the Prisma client will be pointing at the last partner product database until the client is regenerated.
```

Comment:
> When doing this, if you switch partner products, the Prisma client will be pointing at the last partner product database until the client is regenerated.

@garrettdimon As far as I can tell, this is only an issue when developing and running a data migration script. This level of detail might make more sense in a section that talks about data migrations.

Reply:
I feel (not super strongly) that this is a red herring, and that the step here is to `validate the database_url` 

The Prisma client should generally be generated from the root of the monorepo using `pnpm -- turbo run generate` (which also runs with `turbo run build` as a dep) and when it's run in that way it uses a throw-away `database_url`